woocommerce/woocommerce-android
# Issue/5710 Preparation for purchase reader in Canada/other countries

Part of: #5710 


### Description

Passes country to code to the "Hub" screen. It most probably will be used not only for the card purchase URLs

@malinajirka @AnirudhBhat, I'm passing the country code from the onboarding flow, but we can get it from the DB right on the Hub screen. Ideally, we need to add a loading state then and repository. Wdyt what option is better?

### Testing instructions

* Open card reader hub
* Try to click on "purchase a reader."
* Notice a URL used for both cases when IPP Canada is ON and OFF
